packages/kolibri-tools/package.json

```diff
@@ -17,6 +17,7 @@
     "@babel/core": "^7.9.6",
     "@babel/plugin-transform-runtime": "^7.9.6",
     "@babel/preset-env": "^7.9.6",
+    "@testing-library/jest-dom": "^5.11.0",
```

Comment:
I installed this package to be able to get rid of `isVisible` as recommended [here](https://vue-test-utils.vuejs.org/upgrading-to-v1/#isvisible). I am not sure what's our packages release process - do we need to raise `kolibri-tools` version?

By the way the library [looks very useful](https://github.com/testing-library/jest-dom#table-of-contents) - if there's interest maybe we can check it out and start using it for our tests. I can mention it on dev meeting.

Reply:
Yeah, this is a great addition! I don't believe we need to re-publish `kolibri-tools` if we're only using `testing-library` in Kolibri.